pwa/src/components/object_entities/dataTable.tsx

```diff
@@ -29,18 +29,9 @@ export default function DataTable({ id }) {
         Authorization: "Bearer " + sessionStorage.getItem("jwt"),
       },
     })
-      .then((response) => {
-        if (response.ok) {
-          return response.json();
-        } else {
-          setAlert(null);
-          setAlert({ type: 'danger', message: response.statusText });
-          throw new Error(response.statusText);
-        }
-      })
+      .then((response) => response.json())
```

Comment:
Hier wordt `response.json()` returned, maar volgens mij wordt er nergens iets met deze value gedaan.

Ook hebben we twee `.then()` functies wat denk ik naar één omgeschreven kan worden (wel eerst even kijken of we überhaupt de eerste return nodig hebben).